src/components/numbers/Numbers.js

```diff
@@ -0,0 +1,31 @@
+import './numbers.scss';
+
+export const Numbers = (props) => {
+  const {
+    from,
+    to,
+    odd,
+    even
+  } = props;
+  let list = [];
+  for (let i = from; i <= to; i++) {
+    list.push(i);
```

Comment:
i would check here `odd` and `even` instead of doing it with `.filter()` further